tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "allowJs": false,
+    "allowJs": true,
```

Comment:
ktore _javascript_ subory chceme kompilovat?

Reply:
ak uz ich dovolime, nechces zapnut _checkJs_?

Reply:
ak to zmazem, CRA mi to forne na true. Ok, necham false.